packages/react/src/index.ts

```diff
@@ -64,13 +65,39 @@ function setCurrentUpdater(updater?: Effect) {
 	finishUpdate = updater && updater._start();
 }
 
-function createUpdater(update: () => void) {
+/**
+ * A redux-like store whose store value is a positive 32bit integer (a 'version') to be used with useSyncExternalStore API.
+ * React (current owner) subscribes to this store and gets a snapshot of the current 'version'.
+ * Whenever the 'version' changes, we tell React it's time to update the component (call 'onStoreChange').
+ *
+ * How we achieve this is by creating a binding with an 'effect', when the `effect._callback' is called,
+ * we update our store version and tell React to re-render the component ([1] We don't really care when/how React does it).
+ *
+ * [1]
+ * @see https://reactjs.org/docs/hooks-reference.html#usesyncexternalstore
+ * @see https://github.com/reactwg/react-18/discussions/86
```

Comment:
Maybe this one is more relevant 😅 
```suggestion
 * @see https://github.com/reactjs/rfcs/blob/main/text/0214-use-sync-external-store.md
```